app/utils/lava_log_parser.py

```diff
@@ -105,8 +155,12 @@ def run(log, boot, txt, html):
         "Boot result": boot_result_html,
         "Errors": numbers["error"],
         "Warnings": numbers["warning"],
+        "Kernel Panic": kernel_numbers["0"],
+        "Kernel Errors": kernel_numbers["3"],
+        "Kernel Warnings": kernel_numbers["4"]
     }
-    for title, value in results.iteritems():
+
```

Comment:
Out of interest, why use `.items()` rather than `.iteritems()`?

As far as I know, `.items()` returns a list of (key, value) pairs whereas `.iteritems()` doesn't create any intermediate list but an iterator to directly iterate over the items.  So if having a list is not useful, then `.iteritems()` should be faster and use less memory (probably not noticeable in this case, but in principle this can make a difference with large dictionaries).

Reply:
@gctucker good point! I took something properly reading the documentation..and I saw that we don't have an equivalent `iteritems` in python 3. Reading a little bit more...I saw that in some cases it's interesting to import a lib to use `iteritems` and avoid to use some much memory allocating a list o/

Reply:
Yes, we're still using Python 2.7... ;)